package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "cartoframes",
+  "version": "1.0.0",
+  "description": "*********** CARTOframes ***********",
+  "main": "./cartoframes/assets/src/index.js",
+  "directories": {
+    "doc": "docs",
+    "example": "examples",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "rollup --config --watch",
+    "build": "rollup --config"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CartoDB/cartoframes.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/CartoDB/cartoframes/issues"
+  },
+  "homepage": "https://github.com/CartoDB/cartoframes#readme",
+  "dependencies": {
+    "js-base64": "^2.5.1",
+    "jshint": "^2.10.2",
+    "rollup": "^1.17.0"
+  },
+  "devDependencies": {
+    "Base64": "^1.0.2",
+    "base64-js": "^1.3.0",
+    "rollup-plugin-node-resolve": "^5.2.0"
+  }
```

Comment:
I don't know what is the approach we are following to distinguish between dependencies and dev-dependencies, but for me, jshint should be a dev-dependency and rollup libraries should be classified together.

Reply:
you're right!